app-backend/batch/src/main/scala/export/UpdateExportStatus.scala

```diff
@@ -154,64 +158,25 @@ final case class UpdateExportStatus(
       name: Option[String],
       id: UUID,
       exportType: String
-  ) = {
-    val email = new NotificationEmail
-
+  ): IO[Unit] = {
     (user.getEmail, platform.publicSettings.emailExportNotification) match {
-      case ("", true) =>
```

Comment:
Removing these two branches with `""` for the user email is fine because the email settings validation will reject it